src/monster.cpp

```diff
@@ -429,7 +429,7 @@ int monster::print_info(WINDOW* w, int vStart, int vLines, int column) const
     const auto hp_desc = hp_description( hp, type->hp );
     mvwprintz( w, vStart++, column, hp_desc.second, "%s", hp_desc.first.c_str() );
 
-    std::vector<std::string> lines = foldstring(type->description, getmaxx(w) - 1 - column);
+    std::vector<std::string> lines = foldstring( type->get_description().c_str(), getmaxx(w) - 1 - column );
```

Comment:
Here you don't need to call `c_str`, `foldstring` works with an `std::string`.

`description` is also used (without translation) in iuse.cpp - you should probably make it private in `mtype`, so your compiler will tell where it's used directly.